### PR TITLE
Bookstore CMS - MILESTONE FIVE UI

### PR DESCRIPTION
**Pull request by christianotieno**
# Milestone 5: UI (see the following steps below for details)
## Dear reviewer, in this milestone, we carried out the following:

- [x] Review the provided design file from to identify all the elements.
- [x] Implement the styling of the different components based on the design.

##
We hereby welcome your timely review, thank you!

### CODE REVIEW

File: src/styles/scss/main.scss [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+.book-div button,
+.book-form button {
+  font-size: 25px;
+  color: #fafafa;
+  margin: 16px 20px;
+  font-weight: bold;
+  text-transform: uppercase;
+  background-color: #0290ff;
```
Comment (bellom) [inline src/styles/scss/main.scss]: you might want to apply the following;
1. `border-style: none`
2. `cursor: pointer`

before::
image


after:: 
image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

You have done the basic, we just need to little more styling 🙏 so that your design can look almost the same as the project spec given:
But please Guys, we have strict guidelines as well to follow and one of the requirements is seeing into it that students replicated requirements.

If many students do this, we'll gradually shift away from the requirement from team to team.

Your design is very good, just not requirement-compliant, please try to adhere.THANKS
That's just why you don't get a thumbs up now.......

image


image


Let's work on the following:
1. I think you might want to reduce the height and the width of the `remove` button or kindly make use of the style from the original webpage provided by Microverse.
2. The `book `form` (you need to fix the height of the category and kindly make it similar to the original web page.) 
3. Kindly working on responsiveness, that's is; mobile view; category dropdown menu seems disappeared.
image


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED

Hi @christianotieno @collinsugwu good job team :+1:  everything is almost done except for a little design issue
review

From the screenshot above the "add book" button is not wide enough and also according to the original design, all the buttons have rounded edges. 

Please look into that :+1:  and request for another review :slightly_smiling_face: 

**RAPHAEL NORIODE** :mask:

**Review by AdnanAfsari — APPROVED** [review]
**Status: Approved**

Hello @christianotieno @collinsugwu ✋ 

Nice improvement,Looks good now so it's a 👍  from me.

Happy Coding
Adnan Afsari

**Pull request merged**